loadtest/spec.go

```diff
@@ -5,13 +5,17 @@ import (
 )
 
 const (
+	// ApdexPassThreshold is the minimum value allowed for the test to pass.
```

Comment:
I would like to have more info on what this means and is about and why the number is choose. And what does apdex mean?

Reply:
It's in the wall of text in the PR description ;) https://docs.stormforger.com/faq/#what-is-apdex

It's basically a metric for user satisfaction. It's quite widely supported and I used to use it back in the day with New Relic.

For this PR it's just because I want something simple to test. I want to get the automation sorted so we have a full load test using StormForger controlled from a Circle job. 

The test that is executed is in this Helm chart. Sadly it's in JavaScript ;) Once this is merged we can start iterating on the tests. End goal is to have confidence to enable HPA for Ingress Controller in all production clusters.

https://github.com/giantswarm/stormforger-cli/blob/1c086b187dc27c3ef979c48a07f1d6818855f08a/helm/stormforger-cli-chart/templates/configmap.yaml#L28-L30

Reply:
Having info in the PR does not help when people work on the code. When we can simply add the link to the godoc this would be already super useful.